src/views/v2/commerce/gateway/stripe/card_element.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/buttons.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ * @var bool $must_login [Global] Whether login is required to buy tickets or not.
+ */
+
+if ( $must_login ) {
+	return;
+}
+?>
+<div id="tec-tc-gateway-stripe-card-element">
```

Comment:
Let's maybe leave a `@todo` so that Mike can revisit this in the future and see if we use ID or we go with classes.